src/actions/participants.js

```diff
@@ -1,30 +1,30 @@
-import { createAction } from 'redux-actions';
+import { createActions } from 'redux-actions';
 import actionTypes from '../constants/actionTypes';
 import random from 'lodash/random';
 
-const setSelectedParticipant = createAction(actionTypes.SET_SELECTED_PARTICIPANT);
+const actions = createActions({
+    [actionTypes.SET_SELECTED_PARTICIPANT]: (selectedParticipant) => ({ selectedParticipant }),
+    [actionTypes.REMOVE_PARTICIPANT_LIST_SUCCESS]: (listOfParticipants) => (listOfParticipants),
```

Comment:
we can have this `(listOfParticipants) => ({listOfParticipants})` so we don't need to send the curly brances on the action call. same for the below action